tests/installation/logs_from_installation_system.pm

```diff
@@ -46,10 +46,12 @@ sub run {
     if (get_var('BACKEND', '') =~ /ipmi|spvm/) {
         use_ssh_serial_console;
         # set serial console for xen
-        set_serial_console_on_vh('/mnt', '', 'xen') if (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen'));
-        set_serial_console_on_vh('/mnt', '', 'kvm') if (check_var('HOST_HYPERVISOR', 'kvm') || check_var('SYSTEM_ROLE', 'kvm'));
-        adjust_for_ipmi_xen('/mnt')                 if (get_var('REGRESSION') && (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen')));
-        set_pxe_efiboot('/mnt')                     if check_var('ARCH', 'aarch64');
+        if (is_sle) {
+            set_serial_console_on_vh('/mnt', '', 'xen') if (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen'));
+            set_serial_console_on_vh('/mnt', '', 'kvm') if (check_var('HOST_HYPERVISOR', 'kvm') || check_var('SYSTEM_ROLE', 'kvm'));
+            adjust_for_ipmi_xen('/mnt')                 if (get_var('REGRESSION') && (get_var('XEN') || check_var('HOST_HYPERVISOR', 'xen')));
+            set_pxe_efiboot('/mnt')                     if check_var('ARCH', 'aarch64');
+        }
```

Comment:
I don't understand why we don't set serial console on openSUSE.

Reply:
We set serial console in openSUSE, but not here. there is no kvm/xen role creation during installation. We install kvm/xen patterns via zypper, after that serial console is set.

Reply:
OK, can you then please reuse this code on the other place where it is needed in openSUSE? And can you please create a comment about this?

Reply:
Yes, set_serial_console_on_vh is called in switch_to_ssh_and_install_hypervisor.pm which runs before host reboot.
zypper_call("in -t pattern ${hypervisor}_server ${hypervisor}_tools", 1800);
set_serial_console_on_vh('', '', $hypervisor);

comment added.

Reply:
The suggestion from pdostal to reuse the code in both places is still open

Reply:
The code is reused in switch_to_ssh_and_install_hypervisor.pm, see my comment above.

>Yes, set_serial_console_on_vh is called in switch_to_ssh_and_install_hypervisor.pm which runs before host reboot.
>zypper_call("in -t pattern ${hypervisor}_server ${hypervisor}_tools", 1800);
>set_serial_console_on_vh('', '', $hypervisor);